pages/known-issues.md

```diff
@@ -0,0 +1,4 @@
+# Known issues
+
+- The ATC checks are not RFC-enabled ([#268](https://github.com/SAP/code-pal-for-abap/issues/268)).
+- The links to the GitHub documentation in the ADT Problem view don't work in some system releases. This can be fixed by installing SAP Note [3313518](https://launchpad.support.sap.com/#/notes/3313518).
```

Comment:
```suggestion
- The links to the GitHub documentation in the ADT Problems view don't work in some system releases. This can be fixed by installing SAP Note [3313518](https://launchpad.support.sap.com/#/notes/3313518).
```